themes/red-theme/lms/static/sass/partials/bootstrap/_theme.scss

```diff
@@ -0,0 +1,13 @@
+// Sample red theme to demonstrate SASS overrides
```

Comment:
Agreed on this. I'll work on getting the old Red theme to look more similar to this page. I do like the changes we made to get rid of the big bold red header, as it would be hacky to use $brand-primary as the background and $brand-color in the red-theme and the opposite in the standard page.